projects/testing/core/button.harness.ts

```diff
@@ -15,4 +15,8 @@ export class TuiButtonHarness extends TuiComponentHarness {
     public async hasClass(input: string): Promise<boolean> {
         return (await this.host()).hasClass(input);
     }
+
+    public async getSize(): Promise<string | null> {
```

Comment:
Size is never `null`, I think we can add `!` there

Reply:
Sorry I didn't understand `we can add ! there` !  

 I have to enfore `null` because of `getAttribute`  return type. 

`getAttribute(name: string): Promise<string | null>;`

Reply:
`return (await this.host()).getAttribute('data-size')!;`

Reply:
I meant adding `!` at the end. I know it might return `null`, but the button is designed in such a way that this is never possible, so we shouldn't make testing buttons harder by having them to work around `null` there, since it's never going to happen.

Reply:
Or just

```
return (await this.host()).getAttribute('data-size') ?? '';
```

Reply:
Got it Done